cedar-policy-validator/src/human_schema/err.rs

```diff
@@ -356,6 +356,8 @@ pub enum ToJsonSchemaError {
     DuplicateContext { loc1: Loc, loc2: Loc },
     #[error("Duplicate {kind} decleration. Action may have at most once {kind} declaration")]
     DuplicatePR { kind: PR, loc1: Loc, loc2: Loc },
+    #[error("Missing {kind} declaration for {name}. Actions must define both a `principals` and `resources` field")]
```

Comment:
Nits (not introduced by this PR): 
* Error messages should start with lower case letters
* Don't use backticks after a colon (so `A: B` instead of ``` A: `B` ```)